src/toolbar/toolbar-title.jsx

```diff
@@ -60,7 +73,7 @@ const ToolbarTitle = React.createClass({
     }, style);
 
     return (
-      <span style={styles} {...other} >{text}</span>
+      <span {...other} className={className} style={styles}>{text}</span>
```

Comment:
no, style is the raw one, styles is the prepared styles. I'll rename to fix the confusion.